container/view.go

```diff
@@ -313,7 +317,7 @@ func (v *memdbView) transform(container *Container) *Snapshot {
 		Managed:      container.Managed,
 		ExposedPorts: make(nat.PortSet),
 		PortBindings: make(nat.PortSet),
-		Health:       container.HealthString(),
+		Health:       health,
```

Comment:
I believe this was the only place that `(s *State) HealthString()` was called, so please remove that method.